### PR TITLE
Issue 113 merge private GridConnectionCost into public GridConnectionCost module

### PR DESCRIPTION
**Pull request by parangat94**

### CODE REVIEW

**Comment by parangat94** [comment]
@akey7 @eberlea ready for review.

File: landbosse/model/GridConnectionCost.py [hunk 1]
```diff
@@ -63,15 +63,35 @@ def calculate_costs(self, calculate_costs_input_dict, calculate_costs_output_dic
...
+                                                                                                            calculate_costs_input_dict[
+                                                                                                                'num_turbines'] *
+                                                                                                            calculate_costs_input_dict[
+                                                                                                                'turbine_rating_MW'] * 1000) ** (
+                                                                                                        -0.272))
+            calculate_costs_output_dict['trans_dist_usd'] = calculate_costs_input_dict['num_turbines'] * \
+                                                                 calculate_costs_input_dict[
+                                                                     'turbine_rating_MW'] * 1000 * \
+                                                                 calculate_costs_output_dict[
+                                                                     'tower_to_point_of_interconnection_usd_per_kw']
```
Comment (akey7) [inline landbosse/model/GridConnectionCost.py]: It is hard to read this code when lines are split like
```
dictionary[
    'key'] * X
```

Consider breaking out the values out of the keys before you use them to enhance readability:

```
num_turbines = calculate_costs_input_dict['num_turbines']
turbine_rating_MW = calculate_costs_input_dict['turbine_rating_MW']
```

Then use the simpler variable names from these assignments in the main line that calculates `calculate_costs_output_dict['trans_dist_usd']`

File: landbosse/model/GridConnectionCost.py [hunk 2]
```diff
@@ -63,15 +63,35 @@ def calculate_costs(self, calculate_costs_input_dict, calculate_costs_output_dic
...
+                calculate_costs_output_dict['trans_dist_usd'] = 0
             else:
-                calculate_costs_output_dict['interconnect_adder_USD'] = 0
-            calculate_costs_output_dict['trans_dist_usd'] = ((1176 * self.input_dict['interconnect_voltage_kV'] + 218257) * (calculate_costs_input_dict['distance_to_interconnect_mi'] ** (-0.1063)) * calculate_costs_input_dict['distance_to_interconnect_mi']) + calculate_costs_output_dict['interconnect_adder_USD']
+                if calculate_costs_input_dict['new_switchyard'] is True:
+                    calculate_costs_output_dict['interconnect_adder_USD'] = 18115 * self.input_dict['interconnect_voltage_kV'] + 165944
+                else:
+                    calculate_costs_output_dict['interconnect_adder_USD'] = 0
+                calculate_costs_output_dict['trans_dist_usd'] = ((1176 * self.input_dict[
+                    'interconnect_voltage_kV'] + 218257) * (calculate_costs_input_dict['distance_to_interconnect_mi'] ** (
```
Comment (akey7) [inline landbosse/model/GridConnectionCost.py]: See comments about splitting lines below.

**Comment by akey7** [comment]
@parangat94 I just reviewed these changes, and the validation data passes. So go ahead and hold off on splitting the lines up for now. We can go ahead with the code the way it is. Just keep the comments I made earlier as potential thoughts for the future, in case they are helpful.

**Pull request merged**